src/math/Vector3.js

```diff
@@ -721,7 +725,8 @@ Object.assign( Vector3.prototype, {
 
 	}
 
-} );
+}
 
+_vector = new Vector3();
```

Comment:
Would it work to do `const _vector = new Vector3();` here?

Reply:
That works! (I tested)
So, other internal use instances should be declared at the bottom of the file too?

Such as: `const _quaternion = new Quaternion();` in
https://github.com/mrdoob/three.js/blob/f549b793fe070b951fa1018af8cbaddf8650c11f/src/math/Vector3.js#L1-L7
and
https://github.com/mrdoob/three.js/blob/f549b793fe070b951fa1018af8cbaddf8650c11f/src/math/Matrix4.js#L1-L11
also maybe
https://github.com/mrdoob/three.js/blob/f549b793fe070b951fa1018af8cbaddf8650c11f/src/math/Euler.js#L4-L9

Reply:
> So, other internal use instances should be declared at the bottom of the file too?

Yes, I think so!

Reply:
Okay, done it with 8bcfccd